server/modules/record/repository/nodeRepository.js

```diff
@@ -165,6 +165,16 @@ const deleteNode = async (surveyId, nodeUuid, client = db) =>
     dbTransformCallback
   )
 
+const deleteNodes = async (surveyId, nodeDefUuids, client = db) =>
```

Comment:
what about calling it deleteNodesByNodeDefUuids?

Reply:
ok